src/main/plugin/iso19139.ca.HNAP/schematron/schematron-rules-multilingual.sch

```diff
@@ -392,22 +410,32 @@
   <sch:pattern>
     <sch:title>$loc/strings/DataIdentification</sch:title>
 
-    <!-- Use Limitation -->
     <!-- Use Limitation -->
     <sch:rule context="//gmd:identificationInfo/gmd:MD_DataIdentification
         |//*[@gco:isoType='gmd:MD_DataIdentification']
         |//*[@gco:isoType='srv:SV_ServiceIdentification']">
 
+      <sch:let name="openLicenseList" value="geonet:openLicenseList($thesaurusDir)"/>
+      <sch:let name="locMsgMain" value="geonet:appendLocaleMessage($loc/strings/*[name() = concat('OpenLicense', $mainLanguageText)], $openLicenseList)"/>
+      <sch:let name="locMsgAlt" value="geonet:appendLocaleMessage($loc/strings/*[name() = concat('OpenLicense', $altLanguageText)], $openLicenseList)"/>
```

Comment:
Shouldn't $openLicenseList be the alt language list?  Seems like you are supplying the same list for both main and alt error message?